docs/BUILD_PIPELINE_DEBUGGING.md

```diff
@@ -0,0 +1,30 @@
+# Build Pipeline Debugging
```

Comment:
was there something in particular you wanted to see in the description? It's not exactly related to performance 🙂